### PR TITLE
[v6r22] Add Clinics to Hospital

### PR DESCRIPTION
**Pull request by VladimirRomanovsky**
It gives possibility to run jobs for different problematic productions at different Hospital sites 
BEGINRELEASENOTES

ENDRELEASENOTES

### CODE REVIEW

File: TransformationSystem/Client/TaskManager.py [hunk 1]
```diff
@@ -820,13 +820,25 @@ def _handleRest(self, oJob, paramsDict):
...
     """
     oJob.setInputDataPolicy('download', dataScheduling=False)
-    hospitalSite = self.opsH.getValue("Hospital/HospitalSite", 'DIRAC.JobDebugger.ch')
+
+    hospitalSite = None
+    hospitalCEs = []
+    if "Clinics" in self.opsH.getSections("Hospital")['Value']:
+      clinics = self.opsH.getSections("Hospital/Clinics")['Value']
+      for clinic in clinics:
+        if int(transID) in [int(x) for x in self.opsH.getValue("Hospital/%s/Transformations"%clinic, [])]:
```
Comment (fstagni) [inline TransformationSystem/Client/TaskManager.py]: This line and the next(s), just fix the spacing:
```suggestion
        if int(transID) in [int(x) for x in self.opsH.getValue("Hospital/%s/Transformations" % clinic, [])]:
```

File: TransformationSystem/Client/TaskManager.py [hunk 2]
```diff
@@ -559,7 +559,7 @@ def __prepareTasksBulk(self, transBody, taskDict, owner, ownerGroup, ownerDN):
     oJob.setJobGroup(transGroup)
 
     if int(transID) in [int(x) for x in self.opsH.getValue("Hospital/Transformations", [])]:
-      self._handleHospital(oJob)
+      self._handleHospital(oJob,transID)
```
Comment (chrisburr) [inline TransformationSystem/Client/TaskManager.py]: The tests will also fail due to the missing space here and on the instance of _handleHospital
```suggestion
      self._handleHospital(oJob, transID)
```

File: TransformationSystem/Client/TaskManager.py [hunk 3]
```diff
@@ -820,13 +820,25 @@ def _handleRest(self, oJob, paramsDict):
                          transID=transID, method='_handleRest')
           oJob._addJDLParameter(paramName, paramValue)
 
-  def _handleHospital(self, oJob):
+  def _handleHospital(self, oJob, transID):
```
[the remark this answers is not in the document]
Comment (VladimirRomanovsky) [reply TransformationSystem/Client/TaskManager.py]: May be ...

[the remark this answers is not in the document]
Comment (VladimirRomanovsky) [reply TransformationSystem/Client/TaskManager.py]: I suppose should not, as it is internal function of class and it should not call from outside, I hope.

[the remark this answers is not in the document]
Comment (fstagni) [reply TransformationSystem/Client/TaskManager.py]: Still put a default value, please.

**Comment by fstagni** [comment]
@VladimirRomanovsky please finish this PR according to the comments, and then it'll be merged.

**Pull request merged**